src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHighCommon.td

```diff
@@ -57,4 +57,10 @@ def ACT_RELUAttr: NativeCodeCall<"$_builder.getStringAttr(\"ACT_RELU\")">;
 
 def GetTypeOf : NativeCodeCall<"$0.getType()" >;
 
+def GetNullAttr : NativeCodeCall<"Attribute()">;
+
+def GetZeroI64Attr: NativeCodeCall<
+  "IntegerAttr::get($_builder.getIntegerType(64, /*isSigned=*/true), APInt(64, 0, /*isSigned=*/true))"
+>;
```

Comment:
When we have time, we may need have a common.td for all the rewriting.

Reply:
Totally agree, there are many duplicated definitions now.